data/categories.txt

```diff
@@ -16556,3 +16556,44 @@ uk:Соціальна установа
 vi:Cơ sở xã hội
 zh-Hans:社会设施
 zh-Hant:社會設施
+
+emergency-emergency_ward_entrance|@category_hospital
+en:Emergency Ward Entrance
+ar:مدخل جناح الطوارئ
+be:Уваход у аддзяленне хуткай дапамогі
+bg:Вход на спешното отделение
+ca:Entrada d'urgències
+cs:Vchod do pohotovostního oddělení
+da:Indgang til akutafdeling
+de:Eingang der Notfallstation
+el:Είσοδος θαλάμου έκτακτης ανάγκης
+es:Entrada a urgencias
+et:Erakorralise meditsiini osakonna sissepääs
+eu:Larrialdietarako Sarrera
+fa:ورودی بخش اورژانس
+fi:Hätäosaston sisäänkäynti
+fr:Entrée du service des urgences
+he:כניסה למיון
+hi:आपातकालीन वार्ड प्रवेश द्वार
+hu:Sürgősségi kórterem bejárata
+id:Pintu Masuk Bangsal Darurat
+it:Ingresso del reparto di emergenza
+ja:救急病棟入口
+ko:응급 병동 입구
+mr:आपत्कालीन प्रभाग प्रवेशद्वार
+nb:Inngang til akuttmottaket
+nl:Ingang noodafdeling
+pl:Wejście na oddział ratunkowy
+pt:Entrada do Serviço de Urgência
+pt-BR:Entrada da ala de emergência
+ro:Intrarea secției de urgență
+ru:Вход в отделение неотложной помощи
+sk:Vchod do pohotovostného oddelenia
+sv:Ingång till akutmottagning
+sw:Mlango wa Wadi ya Dharura
+th:ทางเข้าแผนกฉุกเฉิน
+tr:Acil Servis Girişi
+uk:Вхід до відділення невідкладної допомоги
+vi:Lối vào khu cấp cứu
+zh-Hans:急诊室入口
+zh-Hant:急診病房入口
```

Comment:
Honestly, I have doubts here. Please, remove it for now.

Reply:
Removed